packages/clerk-js/src/ui/icons/question.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M13.4001 7.0001C13.4001 10.5347 10.5347 13.4001 7.0001 13.4001C3.46548 13.4001 0.600098 10.5347 0.600098 7.0001C0.600098 3.46548 3.46548 0.600098 7.0001 0.600098C10.5347 0.600098 13.4001 3.46548 13.4001 7.0001ZM7.00011 4.6001C6.705 4.6001 6.44618 4.75947 6.30664 5.00068C6.08541 5.38313 5.59603 5.51382 5.21358 5.29258C4.83113 5.07135 4.70044 4.58197 4.92168 4.19952C5.33544 3.48424 6.11076 3.0001 7.00011 3.0001C8.32559 3.0001 9.40011 4.07461 9.40011 5.4001C9.40011 6.44507 8.73226 7.33406 7.80011 7.66353V7.8001C7.80011 8.24192 7.44194 8.6001 7.00011 8.6001C6.55828 8.6001 6.20011 8.24192 6.20011 7.8001V7.0001C6.20011 6.55827 6.55828 6.2001 7.00011 6.2001C7.44194 6.2001 7.80011 5.84193 7.80011 5.4001C7.80011 4.95827 7.44194 4.6001 7.00011 4.6001ZM7.0001 11.0001C7.44192 11.0001 7.8001 10.6419 7.8001 10.2001C7.8001 9.75827 7.44192 9.4001 7.0001 9.4001C6.55827 9.4001 6.2001 9.75827 6.2001 10.2001C6.2001 10.6419 6.55827 11.0001 7.0001 11.0001Z" fill="black" fill-opacity="0.3"/>
```

Comment:
🔧 
We need to remove `fill-opacity="0.3"` and set `fill` to `currentColor` so this svg is customisable through CSS